cmd/manager/main.go

```diff
@@ -47,8 +98,22 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Delete incompatible ConfigMap lock if it exists
+	lockName := "cluster-storage-operator-lock"
+	errWait := wait.PollImmediate(time.Second*10, time.Minute*5, func() (bool, error) {
+		err := tryDeleteIncompatibleLock(cfg, lockName)
+		if err != nil {
+			return false, err
```

Comment:
Please log the error. 

* `log.Error(errWait, ...` below logs `Deadline exceeded` or some timeout message, not the API error
* It would be better to get some error logged before 5 minute timeout, users are not that patient.

Reply:
Added log